README.md

```diff
@@ -1,73 +1,92 @@
-# Role Name
+# crypto_policies
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This Ansible role manages system wide crypto policies.
+
+This concept is well adopted since Red Hat Enterprise Linux 8 and in Fedora.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+The system wide crypto policies are implemented and tested on RHEL 8/CentOS 8
+and Fedora.
```

Comment:
If I see right, the CI runs only CentOS. Would it be possible to add also Fedora images? Or should I set up Github Actions CI for that?

Reply:
I'm in the process of adding crypto_policies to CI - once this is done, it will be tested on RHEL6, RHEL7, RHEL8 (8.3), RHEL X (8.4 pre-release), CentOS6, CentOS7, CentOS8, Fedora 32, and Fedora 33.

Reply:
Thanks for clarification!